tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -403,3 +403,32 @@ fn test_read_index_when_transfer_leader() {
     );
     drop(cluster);
 }
+
+#[test]
+fn test_local_read_cache() {
+    let mut cluster = new_node_cluster(0, 3);
+    configure_for_lease_read(&mut cluster, Some(50), None);
+    cluster.pd_client.disable_default_operator();
+    cluster.run();
+    cluster.must_put(b"k1", b"v1");
+    must_get_equal(&cluster.get_engine(1), b"k1", b"v1");
+    must_get_equal(&cluster.get_engine(2), b"k1", b"v1");
+    must_get_equal(&cluster.get_engine(3), b"k1", b"v1");
+
+    let r1 = cluster.get_region(b"k1");
+    let leader = cluster.leader_of_region(r1.get_id()).unwrap();
+    let new_leader = new_peer(3 - leader.get_id(), 3 - leader.get_id());
+    cluster.must_transfer_leader(r1.get_id(), new_leader);
```

Comment:
Should trigger a read after became leader.

Reply:
This transfer leader is for we can remove the peer successfully.